### ISSUE TITLE
Tool invocation for extension tools are not announced to screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#266256]
We only know about them in the rendering code, so would need a way to set / check for the ID to ensure it's only announced on initial render

cc @roblourens

**Comment by meganrogge** [comment]
I think it's the change from `alert` to `status` that's causing this

**Comment by alexr00** [comment]
I'm still seeing it not read the tool invocation info until the whole response has come in. Testing with version `97da3098581a5fe4e4d20a86eaff1f6e3ae1a4ae`, and Chat version `0.33.2025110701`. I made sure to enable screen reader mode and reload.

**Comment by meganrogge** [comment]
When I screen record, the screen reader malfunctions. What I see is the status messages coming in as the progress occurs.

**Comment by meganrogge** [comment]
Marking this as verified and will ask @jooyoungseo and @rperez030 to let us know if this works for them as expected.
